lib/assessment.rb

```diff
@@ -43,6 +48,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    file.open("filename.txt") {|f| f.readline}
```

Comment:
When you open a file using `File` the file is read as an collection of ... what? What is each element?